demo/parameter_files/pmp_input_parameters_demo1.py

```diff
@@ -40,7 +45,7 @@
 period = '1979-1989'  # PERIOD OF CLIMATOLOGY
 realization = 'r1i1p1' # REALIZATION
 
-
+save_mod_clims = False 
```

Comment:
not needed anymore, but nice to show them that they can turn it on. We should add a comment on top of it.